.github/workflows/doc.yml

```diff
@@ -44,15 +46,32 @@ jobs:
         sudo apt -y install libfreetype-dev libharfbuzz-dev libfribidi-dev
         sudo apt -y install graphviz
 
-    - name: Build
+    - name: Build (main)
+      run: |
+        ./configure.py --buildtype debug --debug-opts gl vk gpu_capture
```

Comment:
Would it possible to factorize the build args into a env variable to avoid duplication ?

Reply:
After an offline discussion, we will do that later as it requires a new round of tests.